viewer/src/components/MoveButton.vue

```diff
@@ -155,22 +161,25 @@ export default class MoveButton extends Vue {
         this.emitCommand(hex.toString());
       } else {
         this.$store.commit("gaiaViewer/highlightHexes", this.button.hexes);
-        this.subscribeHexClick((hex, highlight) => this.emitCommand(hex.toString(), { warnings: highlight.warnings }));
+        this.subscribeHexClick((hex, highlight) => {
+          if (this.button.needConfirm) {
+            this.$store.commit("gaiaViewer/highlightHexes", new Map<GaiaHex, HighlightHex>([[hex, {}]]));
+          }
```

Comment:
Is this still needed?

Reply:
To see which hex you clicked. A preview of the building would be even better